os/mm/kmm_heap/kmm_memalign.c

```diff
@@ -112,7 +112,11 @@ void *kmm_memalign_at(int heap_index, size_t alignment, size_t size)
 	ret = mm_memalign(&kheap[heap_index], alignment, size);
 #endif
 	if (ret == NULL) {
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
```

Comment:
```
#ifdef CONFIG_...
   ret = mm_memalign(...);
   if (ret == NULL) {
      mm_manage_alloc_fail(...);
#else
...
#endif
```
or
```
ret = mm_memalign(.....
#ifdef CONFIG_..
         , caller_retaddr
#endif
        );
if (ret == NULL) {
    mm_manage_alloc_fail(.....
#ifdef CONFIG_...
                        , caller_retaddr
#endif
                         );
```

Just suggestion.

Reply:
I think .. let us use the second method.